app/views/HomeView.coffee

```diff
@@ -159,7 +159,7 @@ module.exports = class HomeView extends RootView
 
     if me.isAnonymous()
       if document.location.hash is '#create-account' or utils.getQueryVariable('registering') == true
-        @openModalView(new CreateAccountModal())
+        @openModalView(new CreateAccountModal({ origin: 'esports' }))
```

Comment:
Small nitpick. I think we only want to make the origin `esports` in the case that the query variable `registering` is set.

I think this might also be a good opportunity to refactor `registering` into `registering-esports` or something more specific.
Currently the first part of the check `document.location.hash is '#create-account'` will create false positives for the `esports` origin. Whilst only `utils.getQueryVariable('registering') == true` are the users registering from esports.